src/main/resources/org/sagebionetworks/web/client/view/ProfileViewImpl.ui.xml

```diff
@@ -68,6 +68,12 @@
                             ui:field="projectSearchButton"
                             text="Search All Projects"
                             icon="SEARCH"
+                            visibleOn="SM_MD_LG"
+                          />
+                          <b:Button
+                            ui:field="projectSearchButtonSmall"
+                            icon="SEARCH"
+                            visibleOn="XS"
                           />
```

Comment:
`projectSearchButton` also has a clickhandler configured in ProfileViewImpl.java -- shouldn't this new button also be configured the same way?

Reply:
Thanks, I wasn't aware of this